### ISSUE TITLE
Tasks : extend comments drawer

### ISSUE THREAD

**Issue by Jihed525** [Meeds-io/meeds#2707]
**Is your feature request related to a problem? Please describe.**
This request is a feature enhancement which suggests to be able to extend the tasks comments drawer as done for activities.

eXo feedback id: 75591

**Comment by Jihed525** [comment]
I close this one as already available

**Comment by srenault-meeds** [comment]
@Jihed525 I don't think it is possible for tasks comments. Indeed, this is not the same component used for stream and for tasks
